REQUIRE

```diff
@@ -1,4 +1,4 @@
-julia 0.6-
+julia 0.6
```

Comment:
I believe this is too early. I'm pretty sure Julia master won't satisfy this requirement.